src/vm/method.cpp

```diff
@@ -5404,7 +5404,7 @@ BOOL MethodDesc::HasNativeCallableAttribute()
     return (hr == S_OK);
 #endif //FEATURE_CORECLR
 
-#endif //_TARGET_AMD64_
+#endif //_TARGET_AMD64_ || defined(_TARGET_ARM64_)
```

Comment:
It might be better to also add a NYI or assert when the target isn't ARM64 or X64 as otherwise someone has to track this failure down in the same way that you just did.